computation_graph/graph_test.py

```diff
@@ -906,5 +906,8 @@ def test_unwanted_ambiguity_asserted():
     g = graph.make_standard_edge(
         source=_node1, destination=_node2, key="arg1"
     ), graph.make_standard_edge(source=_node1, destination=_node2)
-    with pytest.raises(AssertionError):
+    with pytest.raises(
+        AssertionError,
+        match="There are multiple edges with the same source, destination, and priority in the computation graph!",
```

Comment:
don't like that it's duplicated... do you have to give a string here?
anyway lgtm